script/utils/fix_imports.py

```diff
@@ -38,14 +39,14 @@ def extract_imports(content: str) -> List[str]:
     # Updated pattern to handle multi-line imports
     import_pattern = r'import\s+.*?;'
     imports = re.findall(import_pattern, content, re.MULTILINE | re.DOTALL)
-    
+
```

Comment:
Sorry I have configured my IDE to remove unused spaces. I've just modified here the `IMPORT_PRIORITY` to add `"valuations"`